core/src/main/java/jenkins/model/ProjectNamingStrategy.java

```diff
@@ -66,11 +66,27 @@ public static DescriptorExtensionList<ProjectNamingStrategy, ProjectNamingStrate
      *            the name given from the UI
      * @throws Failure
      *             if the user has to be informed about an illegal name, forces the user to change the name before submitting. The message of the failure will be presented to the user.
+     * @deprecated Use {@link #checkName(String, String)}
      */
+    @Deprecated  
```

Comment:
Same, checkstyle is unhappy about this

```suggestion
    @Deprecated
```